### ISSUE TITLE
Craft 4: Element query executed before Craft is fully initialized.

### ISSUE THREAD

**Issue by chrismlusk** [solspace/craft3-freeform#582]
**Describe the bug or issue you're experiencing**

My logs keep filling up with `craft\elements\db\ElementQuery::prepare: Element query executed before Craft is fully initialized` warnings. Looking at the stack trace, it appears Freeform is involved. Specifically, when the `initHoneypot` function accesses the user component ([see line 814](https://github.com/solspace/craft-freeform/blob/v4/packages/plugin/src/Freeform.php#L814)).

```
Element query executed before Craft is fully initialized.
/var/www/html/app/vendor/craftcms/cms/src/elements/db/ElementQuery.php:1242
/var/www/html/app/vendor/craftcms/cms/src/db/Query.php:275
/var/www/html/app/vendor/craftcms/cms/src/elements/db/ElementQuery.php:1506
/var/www/html/app/vendor/craftcms/cms/src/elements/User.php:527
/var/www/html/app/vendor/craftcms/cms/src/web/User.php:500
/var/www/html/app/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php:814
/var/www/html/app/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php:296
/var/www/html/app/vendor/craftcms/cms/src/base/Plugin.php:122
/var/www/html/app/vendor/craftcms/cms/src/Craft.php:53
/var/www/html/app/vendor/craftcms/cms/src/services/Plugins.php:949
/var/www/html/app/vendor/craftcms/cms/src/services/Plugins.php:228
/var/www/html/app/vendor/craftcms/cms/src/base/ApplicationTrait.php:1494
/var/www/html/app/vendor/craftcms/cms/src/web/Application.php:105
/var/www/html/app/vendor/craftcms/cms/src/Craft.php:53
/var/www/html/app/vendor/craftcms/cms/bootstrap/bootstrap.php:252
/var/www/html/app/vendor/craftcms/cms/bootstrap/web.php:40
```

[A cursory search](https://www.google.com/search?q=Element+query+executed+before+Craft+is+fully+initialized.&oq=Element+query+executed+before+Craft+is+fully+initialized.) suggests this might be a common issue with other plugins after the Craft 4 upgrade?

**Craft & Plugin Info:**

- Craft Version: 4.3.6.1
- Freeform Version: 4.0.9
- Freeform Edition: Pro

**Comment by 1stevengrant** [comment]
spotted this too in Sentry...it's just a warning and have disabled them.

**Comment by kjmartens** [comment]
Thanks for reporting this. We'll check into it 🙂

**Comment by kjmartens** [comment]
This should be fixed now in Freeform 4.0.14. 🙂
